geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/json/JsonPathFilterFunction.scala

```diff
@@ -0,0 +1,54 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.features.kryo.json
+
+import com.typesafe.scalalogging.LazyLogging
+import org.geotools.filter.FunctionExpressionImpl
+import org.geotools.filter.capability.FunctionNameImpl
+import org.geotools.filter.capability.FunctionNameImpl.parameter
+import org.geotools.filter.expression.PropertyAccessors
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.opengis.feature.simple.SimpleFeature
+import org.opengis.filter.expression.VolatileFunction
+
+class JsonPathFilterFunction extends FunctionExpressionImpl(
+  new FunctionNameImpl("jsonPath",
+    parameter("string", classOf[String]),
```

Comment:
Fixed, changed to 'value' and 'path' respectively since it returns the value at the path, not necessarily json.